lxd/devices.go

```
@@ -133,6 +132,46 @@ func deviceNetlinkListener() (chan []string, chan []string, error) {
 	return chCPU, chNetwork, nil
 }
 
+func parseCpuset(cpu string) []int {
```

This function should also return an error I think.